src/Workspaces/Core/Portable/FindSymbols/SymbolFinder.cs

```diff
@@ -15,7 +15,11 @@ namespace Microsoft.CodeAnalysis.FindSymbols
     public static partial class SymbolFinder
     {
         /// <summary>
-        /// Finds the symbol that is associated with a position in the text of a document.
+        /// Finds the symbol that is associated with a position in the text of a document. If 
```

Comment:
I don't think we should be calling out the special case in the high-level summary here. I think of this method as just "if there's multiple symbols, it does what we think is the right thing". If nothing else, that's the stated contract. The implementation is just a detail.

Even if we do want to document all the "strange" things, we should put those in remarks, not the high-level summary.